keyboards/ckeys/washington/keymaps/default/config.h

```diff
@@ -0,0 +1,19 @@
+/* Copyright 2019 merlin04
```

Comment:
This file can be deleted.

Reply:
Added

Reply:
Sorry, I misread this as "this line can be deleted." I've removed the file in a new commit.